.travis.yml

```diff
@@ -52,35 +45,36 @@ jobs:
           repo: $DEPLOY_SLUG
           condition: '"$PRIOR_VERSION" != "$RELEASE_VERSION"'
     - stage: deploy
+      if: type != pull_request AND (branch = master OR tag is present)
+      name: Deploy to target environment (master branch=dev; tag=release)
       language: python
       python: "3.6"
       env:
-        - JOB="Deploy to release environment"
-      install: skip
+        - TF_CLI_ARGS_apply="-auto-approve"
+      install:
+        - pip install pipenv
+        - make init
+        - make rclone/install terraform/install terragrunt/install
```

Comment:
Is the `terraform/intall` make target needed as well?  I don't see it in the latest Makefile.

Reply:
No, that one comes from the tardigrade-ci Makefile... https://github.com/plus3it/tardigrade-ci/blob/7079df1d273da59bd1516a4a0486c90655c4eb56/Makefile#L96-L104

Reply:
ah